tools/lambda-promtail/lambda-promtail/main.go

```diff
@@ -83,6 +84,12 @@ func setupArguments() {
 		batchSize, _ = strconv.Atoi(batch)
 	}
 
+	print := os.Getenv("PRINT_LOG_LINE")
```

Comment:
Any specific rationale behind naming of this ENV (`PRINT_LOG_LINE`)?. 

I would prefer to name it more descriptive `PROMTAIL_LAMBDA_PRINT_LOG_LINE`.

Reply:
Ah. never mind. I see other envs also doesn't take any prefixes :(